Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -339,7 +384,7 @@ Best practices
 
 *   When creating a new event PHP class, it is recommended to add an
     :php:`Event` suffix to the PHP class, and to move it into an appropriate
-    folder like :php:`Classes/Event` to easily discover events provided by a
+    folder like :php:`Classes/EventListener` to easily discover events provided by a
```

Comment:
Nope, it is about events, so Classes/Event is correct.

Reply:
Oh, very important note. I mixed this up, and actually the examples where missing the "Listener" suffix, too! Addressed this.

I also added a second "best practice" to recommend this.

Reply:
You don't need in the examples the Listener suffix. For example, in my projects I don't use it, the class names are mostly already long, so no need to add "Listener" to it. You can add it, or you can omit it, I think, there is no best practise.